deltachat-jsonrpc/typescript/test/online.ts

```diff
@@ -86,10 +86,7 @@ describe("online tests", function () {
       null
     );
     const chatId = await dc.rpc.createChatByContactId(accountId1, contactId);
-    const eventPromise = Promise.race([
-      waitForEvent(dc, "MsgsChanged", accountId2),
-      waitForEvent(dc, "IncomingMsg", accountId2),
-    ]);
+    const eventPromise = waitForEvent(dc, "IncomingMsg", accountId2);
```

Comment:
Now APPEND of a sync message somehow triggers `MsgsChanged` w/o a chat id (or maybe some events are reordered as a side effect of this PR?), so i needed to change these tests. Anyway, the proper event is `IncomingMsg` because `MsgsChanged` is for "unimportant" messages, i.e. when a notification isn't needed.

Reply:
Seems events are indeed get reordered. Made a fix for these unneeded `MsgsChanged` events: https://github.com/deltachat/deltachat-core-rust/pull/5910